Plugin/Gateway.php

```diff
@@ -103,4 +103,24 @@ public function getIdealIssuers() {
 			return null;
 		}
 	}
+
+	/**
+	 * change api response error to userfriendly error message
+	 *
+	 * @param responseError from api
+	 *
+	 * @return String
+	 */
+	public function getUserFriendlyError($responseError = '') {
+		switch ($responseError) {
+			case 'Not subscribed to money transfer service':
```

Comment:
I would prefer to put these errors message inside a `const`

Reply:
added it as const in the class